api/rpc/stack/stack.go

```diff
@@ -119,7 +126,9 @@ func (s *Server) processService(ctx context.Context, stack *Stack, serv *service
 	request := &service.ServiceCreateRequest{
 		ServiceSpec: serv,
 	}
-	server := service.Service{}
+	server := service.Service{
```

Comment:
Ok, but I had a comment that I was going to be refactoring those helpers into a separate utility and you would be expected to supply the docker socket. That's why I intentionally had it at the file level and not in the service struct. I guess it's fine for now. I don't won't to hold this PR up, but I will be refactoring it again soon.